src/main/java/com/reminiscence/reminiscence/watson/WatsonController.java

```diff
@@ -0,0 +1,72 @@
+package com.reminiscence.reminiscence.watson;
+
+import com.ibm.cloud.sdk.core.service.security.IamOptions;
+import com.ibm.watson.tone_analyzer.v3.ToneAnalyzer;
+import com.ibm.watson.tone_analyzer.v3.model.DocumentAnalysis;
+import com.ibm.watson.tone_analyzer.v3.model.ToneAnalysis;
+import com.ibm.watson.tone_analyzer.v3.model.ToneOptions;
+import com.ibm.watson.tone_analyzer.v3.model.ToneScore;
+import com.reminiscence.reminiscence.journal.Entry;
+import com.reminiscence.reminiscence.journal.EntryController;
+import com.reminiscence.reminiscence.journal.EntryRepo;
+import com.sun.jndi.ldap.EntryChangeResponseControl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class WatsonController {
+
+    @Autowired
+    ToneRepo toneRepo;
+
+
+    @GetMapping("/watson")
+    public void requestTones(Entry entry) {
+        ToneAnalysis toneAnalysis = getToneAnalysis(entry.getBody());
+        List<ToneScore> listTones = getToneList(toneAnalysis);
+        saveTones(listTones, entry);
+    }
+
+    public void saveTones(List<ToneScore> listTones, Entry entry) {
+        for (ToneScore tone : listTones) {
```

Comment:
Rather than doing this in the Watson controller, this would have been better in the entry controller.  Instead, Watson should just have returned tone objects.

That way, the Watson functionality could have been tested in isolation from the other logic. (No database access required).